docs/decisions/0008-lti-1p3-puggable-lti-configuration.rst

```diff
@@ -0,0 +1,77 @@
+PADV-412: LTI 1.3 Pluggable and re-usable LTI configuration
```

Comment:
I think we'd better add some context on why this feature is so important and what it enables for LTI. @kuipumu